docs/gke/enable_iap.sh

```diff
@@ -1,4 +1,19 @@
 #!/bin/bash
+
+# Copyright 2018 Google Inc. All rights reserved.
```

Comment:
We decided to use  "Copyright The Kubeflow Authors", so I think we could replace Google Inc. with Kubeflow Authors

https://github.com/tensorflow/k8s/issues/266#issuecomment-362336546

Reply:
Thanks, I didn't know that. I'll update soon.